pkgs/applications/science/electronics/diylc/wrapper.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
or use the bash from nixpkgs.

Also can we inline this?

Reply:
Done